Mathlib/Data/DFinsupp/Basic.lean

```diff
@@ -2248,12 +2255,8 @@ variable {R S : Type*}
 
 variable [∀ i, Zero (β i)] [∀ (i) (x : β i), Decidable (x ≠ 0)]
 
-@[to_additive (attr := simp)]
-theorem map_dfinsupp_prod [CommMonoid R] [CommMonoid S] (h : R →* S) (f : Π₀ i, β i)
-    (g : ∀ i, β i → R) : h (f.prod g) = f.prod fun a b => h (g a b) :=
-  h.map_prod _ _
-#align monoid_hom.map_dfinsupp_prod MonoidHom.map_dfinsupp_prod
-#align add_monoid_hom.map_dfinsupp_sum AddMonoidHom.map_dfinsupp_sum
+#noalign monoid_hom.map_dfinsupp_prod
+#noalign add_monoid_hom.map_dfinsupp_sum
```

Comment:
Arguably this is what the subscript x is for; you can align the old ones to your new `map_dfinsupp_prod`